src/server/game/Server/Packets/MiscPackets.cpp

```diff
@@ -451,3 +451,71 @@ WorldPacket const* WorldPackets::Misc::Dismount::Write()
 
     return &_worldPacket;
 }
+
+void WorldPackets::Misc::SaveCUFProfiles::Read()
+{
+    uint8 strLen;
+
+    _worldPacket >> Count;
+
+    for (uint8 i = 0; i < Count; i++)
```

Comment:
It's already checked in WorldSession::HandleSaveCUFProfiles, but I can add a check if you want...